core/src/main/java/com/linecorp/armeria/server/docs/EnumValueInfo.java

```diff
@@ -32,24 +35,38 @@
     private final String name;
     @Nullable
     private final String docString;
+    @Nullable
+    private final Integer literalValue;
 
     /**
      * Creates a new instance.
      *
      * @param name the name of the enum value
      */
     public EnumValueInfo(String name) {
-        this(name, null);
+        this(name, null, null);
+    }
+
+    /**
+     * Creates a new instance.
+     *
+     * @param name the name of the enum value
+     * @param literalValue literal value (for example, int value) for the enum value
```

Comment:
This is an `Integer` value so I think we don't need `(for example, int value)`.

Reply:
Question: The word `literal` is right for this? 
In [gRPC](https://developers.google.com/protocol-buffers/docs/proto3) and [Thrift](https://diwakergupta.github.io/thrift-missing-guide/#_enums), they call the value as a `constant` so I was wondering If we need to use that word too. /cc @anuraaga, @trustin

Reply:
How about `intValue`?

Reply:
That's a good idea. `literal` is somewhat unclear to understand for me. 😉

Reply:
I'm with you!